docs/components/sensor/bme280.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Configure a bme280 sensor"
+linkTitle: "bme280"
+weight: 70
+type: "docs"
+description: "Configure a bme280 sensor."
+tags: ["sensor", "components"]
+icon: "img/components/sensor.png"
+# SME: #team-bucket
+---
+
+Configure a `bme280` sensor to integrate a [BME280 environmental sensor](https://www.adafruit.com/product/2652), used for sensing temperature, humidity, and barometric pressure, into your robot.
+
+Configure a `bme280` sensor as follows:
```

Comment:
```suggestion
Configure a `bme280` sensor to integrate a [BME280 environmental sensor](https://www.adafruit.com/product/2652), used for sensing temperature, humidity, and barometric pressure, into your robot:
```

^ I think that would be a good change for all the pages. It's otherwise a bit duplicative